Apollo.podspec

```diff
@@ -24,7 +24,7 @@ Pod::Spec.new do |s|
   CMD
 
   s.subspec 'Core' do |ss|
-    ss.source_files = 'Sources/Apollo/*.swift','Sources/ApolloAPI/*.swift'
+    ss.source_files = 'Sources/Apollo/*.swift','Sources/ApolloAPI/**/*.swift'
```

Comment:
Do we need to add the `**/*` to Apollo also? You've only added it to ApolloAPI

Reply:
Not unless we start better organizing the Apollo folder source files into subdirectories. Right now they're all in one folder and the only subfolder in there is `Documentation.docc` which won't contain any source files. So we're fine for now but might need to later.